Modules/Misc/CommunityMedals.cs

```diff
@@ -464,6 +466,23 @@ static void PostSetLevel(LevelInfo __instance, LevelData level)
                 }
                 else
                     __instance.devStamp.SetActive(false);
+
+                if(hideOld.Value)
+                {
+                    String hiddenTime = "?:??.???";
+                    if(medalEarned < I(MedalEnum.Sapphire))
+                        __instance._aceMedalTime.text = (string)styleTime.Invoke(__instance, [
```

Comment:
`LevelInfo.StyleMedalTime` with `achieved` passed as false just passes the string right back. You can get away with just setting the string directly for cleanliness.
![image](https://github.com/user-attachments/assets/9e7c2192-cbd6-40a0-90f0-8e39701c9045)